web/less/room.less

```diff
@@ -223,10 +223,9 @@ body {
       display: none;
     }
 
-     @media @iphoneAll @Landscape {
-             width: 2px;
-             min-width: 2px;
-     }
+    @media @iphoneAll @Landscape {
+      display: none;
+    }
```

Comment:
I don't understand this change.

Reply:
this is to hide the audio switch  in landscape mode which was also hiding in portrait mode

Reply:
Why do we set `display: none;` for `@media @smartphonesPortrait` and `@media @iphoneAll @Landscape` as opposed to, say, `@media @smartphonesPortrait` and `@media @smartphonesLandscape`? Why do we have separate variables for `@smartphones` and `@iphone`?